test_pytest_html.py

```diff
@@ -202,6 +202,31 @@ def pytest_runtest_makereport(item, call):
             href)
         assert link in html
 
+    @pytest.mark.xfail(
+        sys.version_info < (3, 2),
+        reason='Only affects Python versions later than 3.2',
+        run=False)
+    def test_extra_bytes(self, testdir):
+        content = bytes(str(random.random()), 'utf-8')
```

Comment:
This will give you a bytes-object with a random number in it - not sure how useful that is? :wink:

Also, the more idiomatic way to do this would be `str(random.random()).encode('utf-8')` - you'll get the same thing, but `.encode()`/`.decode()` are more common than the two-argument form of `bytes`/`str`